inst/tinytest/test_deserialization.R

```diff
@@ -23,18 +23,18 @@ simplify_lvl <- list(
 # scalar =======================================================================
 #* integer ---------------------------------------------------------------------
 test <- "1"
-target <- 1L
+target <- 1
 expect_identical(
   RcppSimdJson:::.deserialize_json(test),
   target
 )
-if (requireNamespace("bit64", quietly = TRUE)) {
-    target <- bit64::as.integer64(target)
-    expect_identical(
-        RcppSimdJson:::.deserialize_json(test, int64_r_type = int64_opt$always),
-        target
-    )
-}
+#if (requireNamespace("bit64", quietly = TRUE)) {
```

Comment:
I sometimes wrap `if (FALSE) {` and `}` around but editor-commenting-out is quick.

What happened to int64_t support?  Can we reenable it?